src/components/DeFi/AssetInput.stories.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable import/no-anonymous-default-export */
+/* eslint-disable import/no-default-export */
+
+import { Container } from '@chakra-ui/react'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+import { AllocationTable } from './components/AllocationTable'
+import { AssetInput, AssetInputProps } from './components/AssetInput'
+
+export default {
+  title: 'Forms/AssetInput',
+  component: AssetInput,
+  decorators: [
+    (Story: any) => (
+      <Container maxWidth='md'>
+        <Story />
+      </Container>
+    ),
+  ],
+} as ComponentMeta<typeof AssetInput>
+
+const Template: ComponentStory<typeof AssetInput> = (args: AssetInputProps) => (
```

Comment:
Since we're just using own props, I'd destructure and use them explicitly but that's fine like this as it's just a story

Reply:
This is just following the recommended TS for storybook -- https://storybook.js.org/docs/react/writing-stories/stories-for-multiple-components